client/src/index.tsx

```diff
@@ -4,10 +4,11 @@ import { ApolloProvider } from 'react-apollo';
 import { ApolloClient } from 'apollo-client';
 import { createHttpLink } from 'apollo-link-http';
 import { InMemoryCache } from 'apollo-cache-inmemory';
+import GlobalFonts from './fonts/fonts';
```

Comment:
I want to separate imports from node_modules with imports from our own modules. That's why there is an empty line between them in order to easily keep tracks of our imports.
You should move these 2 imports down to line 10 below.